feature_flag.md

```diff
@@ -4,38 +4,55 @@
 **[this](https://martinfowler.com/articles/feature-toggles.html)**
 **helpful article on the topic.**
 
-## Best practises
+At Companies House we use feature flagging to enable us to deliver features
+without modifying the current behaviours of the system. This may be because
+the features are still being developed or the functionality
+may only become active at a certain date. Other flags may be
+semi-permanent and my be used to toggle functionality on/off as needed (e.g. in
+response to a live support issue.)
+
+The following documentation details how feature flags should be used and
+managed.
+
+## Best practices
 
 ### Naming
 
 * The feature flag name should be descriptive and tell people what the flag
   does before they look for it in the code.
-* It should include a prefix that is both searchable and relevant. The shorter
-  the prefix the harder this will be to search for in our code base. i.e. `FF`
-  is a poor prefix as it is the hexadecimal for 255 a common number in
-  development. Recommendation is to use `FEATURE_FLAG_` which is already used
-  by a number of our code repositories.
+* Feature flags **must** be prefixed with `FEATURE_FLAG_`.
 * The feature flag should contain a creation date in the following format
   `ddmmyy` as a suffix.
 * Avoid using dots in the name where possible as they cause problems on
   Windows based machines.
 
 #### Examples
 
-* `FEATURE_FLAG_FIVE_OR_LESS_OFFICERS_JOURNEY_21102021`
-* `FEATURE_FLAG_COMPANY_STILL_REQUIRED_11032021`
+* `FEATURE_FLAG_FIVE_OR_LESS_OFFICERS_JOURNEY_211021`
+* `FEATURE_FLAG_COMPANY_STILL_REQUIRED_110321`
 
 ### Usage
 
-* Always remove a feature flag once it becomes redundant, together with the
-  code that uses it. It is recommended to create a story to remove the
-  feature flag at some point in the future when it is introduced.
+#### For work relating to features under development/to be enabled on a date
+
+* The feature flag **must** be removed once the behaviour becomes accepted.
+* You should create a story to remove the feature flag at some point in the
+  future when it is introduced. This ensures the work is visible and is
+  not forgotten about.
+* When adding a new feature flag to an application which already has
+  feature flags, consider whether you can remove the old feature flags
+  applying the principle of leaving things tidier than when you found them
+  (known as the
+  [scouting rule](https://auth0.com/blog/a-scout-approach-to-software-development/)).
+
+#### Feature flags not bound to features under development (i.e. semi-permanent flags)
+
 * Note that some feature flags may not only be used for 'under construction'
   work and have a much longer shelf-life (e.g. those to control access to logic
   implemented for Covid-19) or indeed be permanent.
 * Liaise with support about how the flag should be used.
 * Values for LIVE environments should be specified in Release Notes. Values for
```

Comment:
"Values for LIVE environments should be specified in Release Notes _or the Terraform configuration within the project repository_"

Suggest adding this (but not _italicised_) as for new ECS deployed services the config values are present in 'profile' `vars` files.